src/devtools/tests/Explorer.test.tsx

```diff
@@ -54,4 +55,8 @@ describe('Explorer', () => {
       expect(toggleExpanded).toHaveBeenCalledTimes(1)
     })
   })
+
+  it('when the value is a BigInt', () => {
+    expect(displayValue(BigInt(1))).toBe('"1n"')
+  })
```

Comment:
please put this in it's own `describe('displayValue')` block, and add at least one test that also tests for non-big-int please.